changelog-2023.07.13-10.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index ad5bf20071..789ca3bc20 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -113,6 +113,24 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2023.07.13-10" date="2023-07-13">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>New OpenStreetMap data as of July 10, 2023</li>
++           <li></li>
++         </ul>
++       <p>Map Styles:</p>
++         <ul>
++           <li></li>
```

Comment:
• Fresh OpenStreetMap data as of July 10, 2023
• Use Qt6 instead of Qt5, please report any log errors or issues
• Add preference for larger map font size
• Feature's layer is displayed in the Place Page
• Fixed crash when rendering large objects
• Faster loading of bookmarks and tracks
• Support altitude, OsmAnd, and Garmin colors in imported GPX tracks
• Fixed pedestrian ETA for big slopes
• Prefer tertiary roads to service roads in car routing
• Better Devanagari (Marathi) font

Search
• Improved search results ranking
• Fixed shop categories search
• Fixed airport search results with empty names

Translations:
• Updated Catalan, Czech, Finnish, French, German, Italian, Polish, Spanish, Swedish, and Turkish translations

Styles
• Improve visibility and findability of highway services and rest areas
• Added second-hand category, auction, collectors shops, turnstile & exit
• Fixed underwater tunnels not being visible
• Fixed areal water tunnels being drawn over other areas
• Introduced more detailed railway-rail-* types
• Fixed railway-abandoned/preserved bridges
• Added a bridge structure outline man_made-bridge
• Added community center, conference center, exhibition center, events venue
• Increased shop icons priority
• Added boundary=protected_area

Reply:
Done